index.js

```diff
@@ -30,6 +29,13 @@ class PrometheusMetrics {
         .collectDefaultMetrics({ prefix: 'opossum_', timeout: 5000 });
     }
 
+    if (circuits) {
+      this.add(circuits);
+    }
+  }
+
+  add (circuits) {
+    circuits = Array.isArray(circuits) ? circuits : [circuits];
```

Comment:
what happens when you call add with no circuit/circuits?  Maybe move that `throw new Error` to here?

Reply:
I propose just return the function without parameters

Reply:
I think I agree - the prometheus bits can be initialized without any circuits. And with this PR, circuits could be added later so there is no reason to throw.